app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const { getTopics } = require('./controllers/newsControllers');
+const { handle500Status } = require('./controllers/errorHandlingControllers');
+
+
+const app = express();
+app.use(express.json());
```

Comment:
We do not need to be using express.json yet as we only need this later on when doing posts. Please remove for now. 🙂